library/Zend/Mvc/Service/ServiceListenerFactory.php

```diff
@@ -53,6 +53,7 @@ class ServiceListenerFactory implements FactoryInterface
             'DiStrictAbstractServiceFactory' => 'Zend\Mvc\Service\DiStrictAbstractServiceFactoryFactory',
             'FilterManager'                  => 'Zend\Mvc\Service\FilterManagerFactory',
             'FormElementManager'             => 'Zend\Mvc\Service\FormElementManagerFactory',
+            'Zend\Form\Annotation\FormAnnotationBuilder' => 'Zend\Mvc\Service\FormAnnotationBuilderFactory',
```

Comment:
In my opinion one class defined how FQNC and others in different way is not a good solution